src/mac-now-playing/MacNowPlaying.mm

```diff
@@ -0,0 +1,267 @@
+/*****************************************************************************/
+/*                                                                           */
+/*  Copyright (C) 2025 Yves Ndiaye                                           */
+/*                                                                           */
+/* This Source Code Form is subject to the terms of the Mozilla Public       */
+/* License, v. 2.0. If a copy of the MPL was not distributed with this       */
+/* file, You can obtain one at https://mozilla.org/MPL/2.0/.                 */
+/*                                                                           */
+/*****************************************************************************/
+
+#import <Cocoa/Cocoa.h>
+#import <Foundation/Foundation.h>
+#import <MediaPlayer/MediaPlayer.h>
+#include <fstream>
+#include <iostream>
+#include <istream>
+
+#include <libaudcore/drct.h>
+#include <libaudcore/hook.h>
+#include <libaudcore/i18n.h>
+#include <libaudcore/plugin.h>
+#include <libaudcore/probe.h>
+
+#define EXPORT __attribute__((visibility("default")))
+
+class MacNowPlayingPlugin : public GeneralPlugin
+{
+public:
+    static const char about[];
+    static constexpr PluginInfo info = {N_("Mac Now Playing")};
+
+    constexpr MacNowPlayingPlugin() : GeneralPlugin(info, false) {}
```

Comment:
Enable the plugin by default also at runtime, so please use `true` as parameter here.